src/moonlink/src/storage/mooncake_table.rs

```diff
@@ -1164,23 +1167,37 @@ impl MooncakeTable {
             .unwrap();
     }
 
-    /// Spawns a background task to persist and truncate WAL if needed.
-    /// Returns true if there was a task spawned, false otherwise.
+    /// Uses the wal_persistence_metadata and iceberg LSN from the latest iceberg snapshot
```

Comment:
let's use 
- either both english words `wal persistence metadata` and `iceberg LSN`, 
- or both variable / type name, like `[wal_persistence_metadata]` and `iceberg_flush_lsn`